assets/templates/header.inc.html

```diff
@@ -17,7 +17,7 @@
                     <li class="source_header_nav_i"><a class="source_a_l" href="#777">Link</a></li>
                 </ul>
             </nav>
+            <a class="sjs-login"></a>
```

Comment:
Please don't add anything to header.inc.html at all, append it from js. Leaving like this, will only force to do more stuff during the migration, and we won't lose much if we just prepend it.